package.json

```
@@ -22,59 +22,56 @@
     "check": "dependency-check . --no-dev --ignore-module patch-package --ignore-module websocket-reconnector",
     "coverage": "nyc --lines 95 --functions 95 --branches 95 --reporter=lcov --reporter=text npm test",
     "lint": "eslint --cache .",
-    "precommit": "npm run lint",
-    "prepare": "patch-package",
-    "prepublishOnly": "npm run precommit && npm run prepush && tag-matches",
-    "prepush": "npm run check && npm test",
+    "prepare": "patch-package && husky install",
+    "prepublishOnly": "npm run lint && npm run check && npm test && tag-matches",
     "test": "mocha"
   },
   "dependencies": {
-    "axios": "0.18.1",
-    "axios-cookiejar-support": "^0.4.2",
+    "axios": "0.21.1",
+    "axios-cookiejar-support": "^1.0.1",
     "debug": "^4.1.1",
-    "ethereumjs-wallet": "^0.6.3",
+    "ethereumjs-wallet": "1.0.1",
```

Isn't it convenient to keel this dependency range open? The only exceptions should be the 0.x versions as those are not following semver.

```suggestion
    "ethereumjs-wallet": "^1.0.1",
```

Same comment for other fixed deps below.

---

I wasn't sure which is the policy for fixed or open range for deps. I thought you were fixing all of them (I saw that in other repos as well) so I went for consistency. I favor open ranges as well.

---

Applied here https://github.com/autonomoussoftware/metronome-wallet-core/pull/34/commits/8c7b5f456fa84c9b72b2890430d8c7a78794b535 after we discussed it in meet 😅

---

@gndelia we had a problem with `socket.io-client` and had to fix it to `2.1.1`. In fact I fixed it in #23 after making this same mistake. See #9 for details.

---

rollbacked for that dependency here https://github.com/autonomoussoftware/metronome-wallet-core/pull/34/commits/1d1f280276f23e2acced4b004df8d3c6422153ef

it seems it's working with newer versions, but we might want to wait until we releases all the apps and have mobile up to date to try the newer version of this